packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -133,8 +127,8 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
 
       sliceToWorld = (point: Types.Point2) => {
         const ijkPoint = vec3.transformMat4(
-          [0, 0, 0],
-          [point[0], point[1], 0],
+          vec3.create(),
+          vec3.fromValues(point[0], point[1], 0),
```

Comment:
I used normal arrays in this case and a few lines above because I've measured the performance in the past and converting a bunch of points was way slower when creating new vectors with `vec3.create()` because it allocates a new buffer for every point. Just make sure this does not degrades the performance.